README.md

```diff
@@ -188,6 +188,25 @@ sudo xbps-install -S SDL2-devel SDL2_image-devel SDL2_net-devel alsa-lib-devel \
                      speexdsp-devel libatomic-devel libXi-devel
 ```
 
+``` shell
+# Nix 
+# With Home Manager on home.nix (Recommended Permanent Installation)
+home.packages = [ pkg-config gcc_multi cmake ccache SDL2 SDL2_image SDL2_net \ 
+                  fluidsynth glib gtest libGL libGLU libjack2 libmt32emu libogg \ 
+                  libpng libpulseaudio libslirp libsndfile meson ninja opusfile \
+                  libselinux speexdsp stdenv alsa-lib xorg.libXi irr1 ]
+
+# With the standard NixOS Configuration on configuration.nix
+environment.systemPackages = [ 
+                  pkg-config gcc_multi cmake ccache SDL2 SDL2_image SDL2_net \ 
+                  fluidsynth glib gtest libGL libGLU libjack2 libmt32emu libogg \ 
+                  libpng libpulseaudio libslirp libsndfile meson ninja opusfile \
+                  libselinux speexdsp stdenv alsa-lib xorg.libXi irr1 
+                  ]; 
```

Comment:
Suggest de-duping the big list of packages so there's only one list.

If home manager is the recommended approach for how to compile code on NixOS, then that example can stay as-is. 

Then for the non-standard `environment.systemPackages` approach, mention that the same list of packages applies.

Reply:
I agree